config/constants/settings/definition.rb

```diff
@@ -102,6 +102,11 @@ class Definition
         description: "Time in minutes to wait before uploaded files not attached to any container are removed",
         default: 180
       },
+      antivirus_scan_available: {
+        description: "Virus scanning option selectable in the UI",
+        default: true,
+        writable: false
```

Comment:
[nitpicking] The `writable: false` wouldn't strictly be necessary. There is no UI for it anyway so it has no real effect other than preventing somebody from setting it via the console. If ever this setting makes it to the UI it will be a little bit harder to find out why the checkbox is disabled.